coalib/output/ConsoleInteraction.py

```diff
@@ -312,16 +314,30 @@ def print_results_formatted(log_printer,
                                         column=None,
                                         end_column=None,
                                         severity_str=severity_str,
+                                        source_lines=None,
                                         **result.__dict__))
                 continue
 
             for range in result.affected_code:
+                source_lines = None
+
+                if args[0] and (
+                        range.start.line and range.end.line):
+                    source_lines = []
+                    file = range.start.file
+                    idx = int(range.start.line)
+
+                    while idx <= int(range.end.line):
```

Comment:
Oh..I forgot about `extend`. I though of first one before, but it couldn't be possible because of [this](https://github.com/coala/coala/issues/3398). thanks :+1: